### PR TITLE
Release 11.4

### PR DESCRIPTION
**Pull request by erz4**

### CODE REVIEW

File: aws/PAS-Component-Single-Deployment.yaml [hunk 1]
```diff
@@ -732,31 +734,22 @@ Resources:
...
                 hostname $1
-
                 sed -i "s/localhost\\./$1./g" /etc/hosts
-
                 sed -i "s/localhost /$1 /g" /etc/hosts
-
                 chmod 646 /etc/sysconfig/network
-
-                sed -i "s/HOSTNAME=localhost.localdomain/HOSTNAME=$1/g"
+                sed -i "s/HOSTNAME=localhost.localdomain/HOSTNAME=$1/g" \
```
Comment (yogevh) [inline aws/PAS-Component-Single-Deployment.yaml:743]: why did you add   "  \  "
 ?

Comment (erz4) [reply aws/PAS-Component-Single-Deployment.yaml:743]: it's one command in 2 lines

**Comment by mliora** [comment]
@yogevh @shiraberkovich We had a problem with PSMP nested and in our check we noticed that a parameter was missing, so we add the parameter and update it in our azure account and also in the repo.
We didn't want to delete the existing nested so we created a new one with the current version number.
We should decide if we want to keep the version number or to remove it.
I think we can do it on the next release.
